### DISCUSSION TITLE
Paper

### DISCUSSION THREAD

**Discussion by LxytIUON** [Data-Designer/Win_Prediction#2] [General]
A rather interesting research work, where he published?

**Comment by Data-Designer** [comment]
Thank you for your comment. 
You can get the full paper from ACM library.
https://dl.acm.org/doi/abs/10.1145/3485447.3512274

**Comment by Data-Designer** [comment]
Chuang Zhao, Hongke Zhao, Yong Ge, Runze Wu, and Xudong Shen. 2022. Winning Tracker: A New Model for Real-time Winning Prediction in MOBA Games. In Proceedings of the ACM Web Conference 2022 (WWW '22). Association for Computing Machinery, New York, NY, USA, 3387–3395. https://doi.org/10.1145/3485447.3512274
